### PR TITLE
[Form] Add Slider component

### PR DESCRIPTION
**Pull request by MarcoPOtard**

### CODE REVIEW

**Comment by MatouTechnologies** [comment]
This pull request has been deployed to Vercel.
<table>
  <tr>
    <td><strong>✅ Preview:</strong></td>
    <td><a href='https://matou-technologies-purrfect-ui-storybook-14.vercel.app'>https://matou-technologies-purrfect-ui-storybook-14.vercel.app</a></td>
  </tr>
  <tr>
    <td><strong>🔍 Inspect:</strong></td>
    <td><a href='https://vercel.com/matou-technologies/purrfect-ui/EDu5xTg7emTk1ZEf9Jsj2kyE8MRt'>https://vercel.com/matou-technologies/purrfect-ui/EDu5xTg7emTk1ZEf9Jsj2kyE8MRt</a></td>
  </tr>
</table>


<hidden purpose="for-rewritable-pr-comment-action-use" value="deploy-preview-storybook"></hidden>

File: src/form/Slider/Slider.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    }
+    [data-disabled] & {
+      border-color: ${theme.color.primary200};
+      background: ${theme.color.primary200};
+    }
+    top: 50%;
+  `;
+
+  return (
+    <StyledSlider {...props}>
```
Comment (Rcaradec) [inline src/form/Slider/Slider.tsx:102]: `<Title style={{ width: "100%" }}>Slider</Title>`

Comment (Balkeo) [reply src/form/Slider/Slider.tsx:102]: Missing Story title yes but your code suggestion is wrong

Comment (MarcoPOtard) [reply src/form/Slider/Slider.tsx:102]: Title add without style attribut

**Pull request merged**